src/firestore/firestore.ts

```diff
@@ -161,6 +162,21 @@ export class AngularFirestore {
     return new AngularFirestoreCollection<T>(ref, query, this);
   }
 
+  /**
+   * Create a reference to a Firestore Collection Group based on a collectionId
+   * and an optional query function to narrow the result
+   * set.
+   * @param collectionId
+   * @param queryGroupFn
+   */
+  collectionGroup<T>(collectionId: string, queryGroupFn?: QueryGroupFn): AngularFirestoreCollectionGroup<T> {
+    if (major < 6) { throw "collection group queries require Firebase JS SDK >= 6.0"}
```

Comment:
worth throwing a link to the release notes into this error? https://firebase.google.com/support/release-notes/js#version_600_-_may_7_2019

Reply:
Eh, probably not. I imagine we'll be talking about so few users over the next couple months & more advanced ones at that.